### ISSUE TITLE
Solve the rubber-banding issue with 3 or more students in a group. 

### ISSUE THREAD

**Issue by cbradyatinquire** [MathNetDev/MathNetClient#30]
Solution to the rubber banding issue has the flaw that it only works for pair activities.  With 3 or more students, we get XML overwrites on updates, as see in Sep 14 meeting.

**Comment by DinoPuppo** [comment]
Perform the testing only on the test branch for now

**Comment by IsaacNPsmash** [comment]
I tested this using two points with two students and a third student observing (Test branch). I still found that the points rubber-banding when moved at the same time.

**Comment by cbradyatinquire** [comment]
Is the 'fail' of the test here due to our deployment issues?

**Comment by IsaacNPsmash** [comment]
I think so.

**Comment by IsaacNPsmash** [comment]
Tested 10/1/2018 @ 3:08 on Test branch.
I used the same design of two points with two students (mover1 and mover2) and a third student observing (watcher). The rubber banding doesn't happen for either movers or watcher, but a dialog box open for the movers. Specifically, if mover1 moves their point, a dialog box opens for mover2 saying "mathnettest.ucdavis.edu says mover1" and an ok button. Once I hit ok the point moves. This happens for each time mover1 moves their point, so if they are using the buttons as arrow keys, mover2 has to click ok for EVERY time mover1 pressed the button. Note: the student's point is not unselected when hitting the ok button in the dialog box.

**Comment by DinoPuppo** [comment]
Sorry, I forgot to get rid of the alert message; I used it for debugging purposes. I just committed my changes to the test branch, and got rid of this bug. Would you be able to test the rubber-banding issue now?

**Comment by IsaacNPsmash** [comment]
Tested 10/3/18 @ 8:43 pm and 10/4/18 @ 5:46 am on Test branch. Test Failed.
I used the same design of two points with two students (mover1 and mover2) and a third student observing (watcher). The rubberbanding has changed but is not fixed. Something about the motion of the two points and the communication between confuses the sync. (I have found it most reliable to find the bug when clicking the arrow buttons with the mouse, but it happened will all ways to move the points). Example: mover1 controls A=(2,2) by selecting and clicking "Up (l)" button (with the mouse) and then pressing the "Down (k)" button. When mover2 selects B=(-2,-2) and, similarly, clicks "Up (l)" button (with the mouse) and then the "Down (k)" button, point B rubberbands back to B=(-2,-1) only on watcher1's view. From this moment on, watcher1's point B will be a point behind, meaning if mover2 clicks "Down (k)" again then watcher1 will see B=(-2,-3) for a moment and then it will rubberband back to B=(-2,-2). The only way I have found to resync is to log everyone out, refresh the browser, and resend the activity.
Example 2: With only two students (mover1 and mover2, no watcher), the same think happened in the View for the teacher, B would rubberband back to the previous point and then lag back a point from then forward.
